src/database/sql/audio.py

```diff
@@ -298,11 +323,14 @@ def get_emotions(self, audio_id: str) -> str:
         """
 
         query = """
-                SELECT relaxed, relaxed_confidence, party, party_confidence, 
-                       aggressive, aggressive_confidence, happy, happy_confidence, 
-                       sad, sad_confidence, last_updated
-                FROM Audio 
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                SELECT relaxed, relaxed_confidence, party,
+                       party_confidence, aggressive,
+                       aggressive_confidence, happy,
+                       happy_confidence, sad,
+                       sad_confidence, last_updated
+                FROM Audio
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.